.github/workflows/end2end.yaml

```diff
@@ -408,6 +408,19 @@ jobs:
         env:
           STAGE: end2end-http
         if: always()
+      - name: "Debug: SSH to runner"
+        uses: scality/actions/action-ssh-to-runner@1.6.0
+        timeout-minutes: 30
+        continue-on-error: true
+        with:
+          tmate-server-host: ${{ secrets.TMATE_SERVER_HOST }}
+          tmate-server-port: ${{ secrets.TMATE_SERVER_PORT }}
+          tmate-server-rsa-fingerprint: ${{ secrets.TMATE_SERVER_RSA_FINGERPRINT }}
+          tmate-server-ed25519-fingerprint: ${{ secrets.TMATE_SERVER_ED25519_FINGERPRINT }}
+        if: failure() && runner.debug == '1'
+      - name: sleep
+        run: sleep 3600
+        if: failure() && ! cancelled() && runner.debug == '1'
```

Comment:
why do we need the extra wait?

According to the documentation,
> The action pauses the job execution until the user connected to the runner explicitly resume the execution of the job by running the resume command.

So execution will resume either:
* When the timeout expires with no SSH connection
* When the job timeout expires
* When the user explicitely calls resume

Reply:
Indeed won't be needed in that case